### DISCUSSION TITLE
[UI Glitch] Double Header, Inside Activity Section

### DISCUSSION THREAD

**Discussion by MarrieMitsu** [community/community#11719] [Other Feature Feedback, Questions, & Ideas]
I don't think it matters. Somehow additional Header appears in the activity section.

I tried to reproduce it, but has not succeeded. Because it happened when I switched to another program and when I came back it already looked like this

![header_glitch](https://user-images.githubusercontent.com/43091888/154692647-a453d036-7a9d-46cb-81be-bc99cbf044b1.jpg)

When I reload the page it's back to normal

**Comment by gaknoll** [comment]
This is definitely a weird bug @MarrieMitsu - thanks for reporting it! We'll keep an eye out to see if we're also able to reproduce it
